src/pages/home/report/ReportActionCompose.js

```diff
@@ -233,6 +233,7 @@ class ReportActionCompose extends React.Component {
      * @param {String} newComment
      */
     updateComment(newComment) {
+        this.textInput.setNativeProps({text: newComment});
```

Comment:
@aliabbasmalik8 Sorry you ping you this way but I just want to know why did you add this? I am doing some changes to the component and I just want to know was there an issue, that required this. It's pretty urgent we have a couple of deploy blockers.

Reply:
t's used to update comments ASAP because in android there is some issue with the selection position when we update without this.
What issue you are facing?

Reply:
There are multiple things I am working on so I was trying to understand the old code. There is an issue when the cursor jumps. There is an issue when the app crashes......so on.

Reply:
Got it, this code working perfectly and tested as well. you can mention PR here on which you are working so can help.

Reply:
Hmm I could be wrong, but this does not seem like a correct usage of `setNativeProps()`... seems like there are various ways we clear the text input and I'm having a really hard time understanding why... please leave more context in the code in the future.

Reply:
Yes, we can use some other ways as well but you can see in [some other issue conversation](https://github.com/Expensify/App/issues/2920#issuecomment-851836161) reviewer like the way of direct manipulation because uncontrolled input used in the app.